### PR TITLE
fix(producer.rb): report on cluster for produce failures

### PR DESCRIPTION
**Pull request by emaymon**

### CODE REVIEW

File: lib/yotpo_kafka/producer.rb [hunk 1]
```diff
@@ -69,7 +69,7 @@ def handle_produce_failures(topic, error)
                 topic: topic,
                 error: error.message)
       begin
-        RedCross.monitor_track(event: 'produce_failure', properties: { topic: topic })
+        RedCross.monitor_track(event: 'produce_failure', properties: { topic: topic, cluster: @seed_brokers })
```
Comment (omamiya) [inline lib/yotpo_kafka/producer.rb:72]: `@seed_brokers` - is it array? don't you want convert it to string? `@seed_brokers.join(', ')` or something similar ?

File: README.md [hunk 2]
```diff
@@ -13,7 +13,12 @@ gem 'yotpo-ruby-kafka'
 3. Defaults to 127.0.0.1
 
 ###Failure Monitoring
-Define influxdb env vars on your service -INFLUXDB_DB, INFLUXDB_HOST, INFLUXDB_PORT  
+* Consume failures - see http://kafka-failures-ui.us-east-1.yotpo.xyz/
+* Produce failures - build a ELK graph on Grafana:
+```ruby
+Query : log_tag:"yotpo_ruby_kafka" AND message:"Single publish failed"
```
Comment (omamiya) [inline README.md:19]: malik

File: lib/yotpo_kafka/producer.rb [hunk 3]
```diff
@@ -46,7 +46,7 @@ def publish(topic, payload, headers = {}, key = nil, to_json = true)
     def publish_multiple(topic, payloads, headers = {}, key = nil, to_json = true)
       log_debug('Publishing multiple messages',
                 topic: topic,
-                message: value,
+                message: payloads.to_s,
```
Comment (omamiya) [inline lib/yotpo_kafka/producer.rb:49]: dangerous, because the decoding could fail. (protobuffs for example)

Comment (emaymon) [reply lib/yotpo_kafka/producer.rb:49]: by the way until now it had the 'value' as the message - and as you can see there was never a 'value' given on this func

**Review by omamiya — APPROVED** [review]

**Pull request merged**